src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -412,15 +460,14 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
 
     // TODO revisit, use a switch?
     const usecase = determineUsecase(txAux.certificates, signingFiles)
-    if ((usecase === LedgerTypes.TransactionSigningMode.__RESEVED_POOL_REGISTRATION_AS_OPERATOR)
+    if ((usecase === LedgerTypes.TransactionSigningMode.POOL_REGISTRATION_AS_OPERATOR)
       && !isFeatureSupportedForVersion(LedgerCryptoProviderFeature.POOL_REGISTRATION_OPERATOR)
```

Comment:
I am confused that there is no feature for catalyst. Don't we check whether catalyst is supported?

Reply:
Valid point, we can fix it in another PR, related to voting.